docs/tabs.md

```diff
@@ -228,3 +228,30 @@ class-label
 
 class-content
 : Additional CSS classes for the content element.
+
+## Synchronised tab options
+
+### URL parameter
+
+By default, the URL does not indicate the tab that you selected.
+To include the tab as a URL parameter, add a setting like the following example in your `conf.py` file:
+
+```python
+sphinx_design_sync_tabs_url_param = "tab"
```

Comment:
What's the benefit of making this configurable? My general preference is to keep our configuration surface area as small as possible unless somebody explicitly asks for extra functionality. E.g., just force the url param to be something like `sd-last-tab`

Reply:
The benefit of making it configurable is that the URL is customer-facing.  The two common parameters are `code` and `tab`.  My thinking is that the tabs don't always show `code`, so that felt wrong to hard-code.  If you really want it baked in, my preference is `tab` and I can revise it so.

Reply:
my general inclination is always to start with the simplest-humanly-possible configuration/implementation, and only complexify once there's clear demand for it, in order to minimize our maintenance burden and the likelihood of deprecations, given that these projects all have pretty limited ongoing maintenance time.

So I'd suggest we ask this question: how important do you think it is to be able to configure the URL param? If >20% of users would want to configure it, let's add it here. If <20% would, then let's keep it simple and see if people request this in the future.

Reply:
After revising, I forgot one more reason for making the URL search param configurable.  If the end user *does not* set it, then the URL is not set with a search param after clicking a tab (though stickiness still works because local storage is set).  Originally, I thought this was a decent compromise for backward compatibility since the current release doesn't set a URL search param.